projects/packages/videopress/src/client/admin/components/admin-page/libraries.tsx

```diff
@@ -0,0 +1,110 @@
+import { Button, Text } from '@automattic/jetpack-components';
+import { Rect, SVG } from '@wordpress/components';
+import { grid, formatListBullets } from '@wordpress/icons';
+import React, { useState } from 'react';
+import { SearchInput } from '../input';
+import Pagination from '../pagination';
+import VideoGrid from '../video-grid';
+import VideoList from '../video-list';
+import styles from './styles.module.scss';
+import { VideoLibraryProps } from './types';
+
+const LibraryType = {
+	List: 'list',
+	Grid: 'grid',
+} as const;
+
+type LibraryType = typeof LibraryType[ keyof typeof LibraryType ];
+
+const filterIcon = (
```

Comment:
Icons should be located in the [icons folder](https://github.com/Automattic/jetpack/tree/trunk/projects/packages/videopress/src/client/components/icons)